### PR TITLE
Provinces page

### PR DESCRIPTION
**Pull request by BuiPhuongThaob**
## Related Tickets:
- [Task #28244](https://edu-redmine.sun-asterisk.vn/issues/28244)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by BuiPhuongThaob** [comment]
ready

File: resources/views/pages/places.blade.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <div class="row justify-content-center">
+                <div class="col-lg-6">
+                    <div class="section_title text-center mb_70">
+                        <h3>{{ trans('home_page.travel') }}</h3>
+                        <p>{{ trans('home_page.reviewing') }}</p>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                @foreach($provinces as $province)
```
Comment (huyenttm-0901) [inline resources/views/pages/places.blade.php]:
```suggestion
                @foreach ($provinces as $province)
```

File: resources/views/header.blade.php [hunk 2]
```diff
@@ -56,9 +56,15 @@
                             </div>
                         </div>
                         <div class="seach_icon">
-                            <a data-toggle="modal" data-target="#exampleModalCenter" href="#">
-                                <i class="fa fa-search"></i>
-                            </a>
+                            @if (Auth::check())
+                                @if (Auth::user()->avatar == NULL)
+                                    <img src="{{ asset(config('constains.avatar')) }}" 
+                                        id="avt-img-header" alt="User avatar"></label>
```
Comment (huyenttm-0901) [inline resources/views/header.blade.php]: i18n

File: resources/lang/en/home_page.php [hunk 3]
```diff
@@ -3,6 +3,6 @@
         'travel' => 'Travel inspiration',
         'reviewing' => 'Articles - unique moments',
         'America' => 'America',
-        'Places' => '10 Places',
+        'Places' => ' Places',
     ];
```
Comment (huyenttm-0901) [inline resources/lang/en/home_page.php:7]: viết file vi nữa e

**Comment by BuiPhuongThaob** [comment]
ready

File: resources/lang/vi/home_page.php [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?php
+    return [
+        'travel' => 'Cảm hứng du lịch',
+        'reviewing' => 'Những bài đăng - Những khoảnh khắc độc đáo',
+        'America' => 'America',
```
Comment (hoangminhuet) [inline resources/lang/vi/home_page.php:5]: vietsub lại đi chị :v

**Pull request merged**